tests/conftest.py

```diff
@@ -170,6 +170,20 @@ def processor_config(tmp_path_factory):
     return processor_config
 
 
+@pytest.fixture(scope="session")
+def processor_config_offline_chat(tmp_path_factory):
+    processor_dir = tmp_path_factory.mktemp("processor")
+
+    # Setup conversation processor, if OpenAI API key is set
```

Comment:
nit: comment doesn't match code change
```suggestion
    # Setup conversation processor
```